src/profile/ProfilePage.jsx

```diff
@@ -15,6 +15,9 @@ import {
 import { InfoOutline } from '@openedx/paragon/icons';
 import classNames from 'classnames';
 
+import ExtendedProfileFieldsSlot from '../plugin-slots/ExtendedProfileFieldsSlot';
```

Comment:
Bit of a naming nitpick here, how would you feel about this being `AdditionalProfileFieldsSlot` instead of extended? My thought process here is that "additional" more clearly implies "you can use this to add more fields to the proflie" where "extended" may send the impression that existing fields could be modified into "extended" versions.